pkg/operator/bootstrapteardown/bootstrap_teardown_controller.go

```diff
@@ -126,8 +127,7 @@ func (c *BootstrapTeardownController) removeBootstrap() error {
 		return updateErr
 	}
 
-	// check to see if bootstrapping is complete
```

Comment:
don't remove this. Explain what it is better.

Reply:
did not intend to remove it, bad refactor!